stdpopsim/genomes.py

```diff
@@ -153,6 +187,13 @@ class Contig:
     :ivar exclude: If True, ``mask_intervals`` specify regions to exclude. If False,
         ``mask_intervals`` specify regions in keep.
     :vartype exclude: bool
+    :ivar genomic_element_types: a list of stdpopsim.GenomicElementType
```

Comment:
I wasn't sure how to do that properly. Can someone point my to docs that explain how to do it? e.g., how do I reference a submodule? `:class:.ext.`MutationType``?

Reply:
`:class:stdpopsim.GenomicElementType`? Otherwise, I think you can say `:vartype genomic_element_types: typing.List[stdpopsim.GenomicElementType]` and the cross referencing will work.